### PR TITLE
Bug fix: Build not allowed in OnlyDuring/ExceptDuring properties.

### PR DESCRIPTION
**Pull request by splatteredbits**

### CODE REVIEW

File: Whiskey/en-US/about_Whiskey_Tasks.help.txt [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      ProGet.
+    * `SetVariable`: creates a variable that you can use in your whiskey.yml
+      file.
+    * `TaskDefaults`: sets default task properties.
+
+    ## Common Task Properties
+
+    Every Whiskey task has these properties:
+
+    * `OnlyBy`/`ExceptBy`: controls if a task runs if run by a developer or
```
Comment (jeffprice6) [inline Whiskey/en-US/about_Whiskey_Tasks.help.txt]: `controls if a task runs if run` might flow better with something like `controls if a task should run when a build is executed`.

File: Whiskey/en-US/about_Whiskey_Tasks.help.txt [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      the build server. By default, a task always runs. Valid values are 
+      `Developer` or `BuildServer`.
+    * `OnlyDuring`/`ExceptDuring': controls if a task will run in the current
+      run mode. Valid values are `Initialize`, `Build`, and `Clean`. Useful
+      for having custom initialization task, cleaning tasks, etc. 
+      Additionally, each task must opt-in to run during Initialize and Clean 
+      modes. If a task doesn't do anything in either of those modes, these 
+      properties have no affect.
```
Comment (jeffprice6) [inline Whiskey/en-US/about_Whiskey_Tasks.help.txt]: `affect` should be `effect`.

File: Whiskey/en-US/about_Whiskey_Tasks.help.txt [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    * `OnlyOnBranch`/`ExceptOnBranch`: controls what branch a task should run
+      on. By default, a task runs on every branch. When a build is run by a
+      developer, there is no branch information, so tasks that have this 
+      property will never run by a developer. Wildcards are supported.
+    * `WorkingDirectory`: the directory in which the command should execute. 
+  By default, this is the directory of the whiskey.yml file.
```
Comment (jeffprice6) [inline Whiskey/en-US/about_Whiskey_Tasks.help.txt]: Indenting is different..

**Comment by splatteredbits** [comment]
@jeffprice6 Typos and grammar fixed. Ready for re-review.

**Pull request merged**